src/ReDate.re

```diff
@@ -107,6 +110,8 @@ let isFuture = date => date->isAfter(Date.make());
 
 let isPast = date => date->isBefore(Date.make());
 
+let isLeap = year => year mod 400 === 0 || year mod 4 === 0 && year mod 100 !== 0;
```

Comment:
this function is used only for internal purposes, I think it should be a part of the `Internal` module 😊